### ISSUE TITLE
FEATURE REQUEST: SPLIT TUNNELING FIX, CHROME EXTENSION, ADBLOCK

### ISSUE THREAD

**Issue by MindfullChest** [amnezia-vpn/amnezia-client#1038]
1) Please make a chrome extension version, or fix split tunneling on pc by adding a white list apps split tunneling(now only black list).
2) Please integrate adblocking features, especially youtube ads, i know it's somehow possible (because of lagom vpn, that doing in even in andoid youtube app)

**Comment by notgurev** [comment]
It has a white list feature already

**Comment by MindfullChest** [comment]
> It has a white list feature already
Nope, white list only on android, not on windows, on windows only blacklist

**Comment by KeshmiOn** [comment]
Windows, client 4.7.0.0. 
Yep, now a whitelist for websites only. Whitelist selection is not available for apps.

**Comment by zappbrannigan34** [comment]
whitelist for apps not exist on windows

**Comment by gman4** [comment]
Yes. Please make app whitelist in windows.

**Comment by katalogoanatom** [comment]
> whitelist for apps not exist on windows

But it doesn't work.

**Comment by shrek-gabe-itch** [comment]
> > whitelist for apps not exist on windows
> 
> But it doesn't work.

then how it works in other clients?

**Comment by skaunov** [comment]
Messing the focus sounds quite risky to degrade the quality. There's dedicated solutions for this. If you're self-hosting it's better to combine those, if you're buying a service then demand the vendor to provide you something which includes both (essentially what the one you mention does).
>     2. Please integrate adblocking features, especially youtube ads, i know it's somehow possible (because of lagom vpn, that doing in even in andoid youtube app)
